org.mwc.debrief.lite/pom.xml

```diff
@@ -81,6 +81,27 @@
   									<ignore></ignore>
   								</action>
   							</pluginExecution>
+  							<pluginExecution>

```

Comment:
Sorry if I'm boring you @ayeshare - but could you explain what this new block does?